pkg/kubectl/describe.go

```diff
@@ -2166,8 +2167,8 @@ func DescribeEvents(el *api.EventList, w io.Writer) {
 		fmt.Fprint(w, "No events.\n")
 		return
 	}
-	sort.Sort(SortableEvents(el.Items))
-	fmt.Fprint(w, "Events:\n  FirstSeen\tLastSeen\tCount\tFrom\tSubobjectPath\tType\tReason\tMessage\n")
+	sort.Sort(events.SortableEvents(el.Items))
```

Comment:
i would prefer `events.Sortable()` here to avoid duplication, but I think I can live with this :-)